babel.config.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env node */
+
+'use strict';
+
+module.exports = {
+  presets: [
+    '@babel/preset-react',
+    [
+      '@babel/preset-env',
+      {
+        targets: {
+          node: 'current',
```

Comment:
Will update so that it is only configured for the test env.

Was following a recommendation here because of error running Jest "ReferenceError: regeneratorRuntime is not defined": https://github.com/facebook/jest/issues/3126#issuecomment-483320742

Reply:
Can you link to that issue in the file for future reference?